OBSDOCS-462: Update logging uninstall docs (#69255)
openshift/openshift-docs: modules/uninstall-loki-operator.adoc

// Module included in the following assemblies:
//
// * logging/cluster-logging-uninstall.adoc

:_mod-docs-content-type: PROCEDURE
[id="uninstall-loki-operator_{context}"]
= Uninstalling Loki

.Prerequisites

* You have administrator permissions.
* You have access to the *Administrator* perspective of the {product-title} web console.

> * If you have not already removed the {clo} and related resources, you must remove references to the LokiStack from the `ClusterLogging` custom resource.

Reviewer: ```suggestion
* If you have not already removed the {clo} and related resources, you have removed references to LokiStack from the `ClusterLogging` custom resource.
```
- Consider changing the wording to make parallel with the other prereqs. Also "must" is already implied by the fact that this is in a "Prerequisites" section.
- Not sure if "the" is needed before LokiStack?

Author: Perfect, thank you!

.Procedure

> . Navigate to the *Administration* -> *Custom Resource Definitions* page. Click *LokiStack*.

Reviewer: ```suggestion
. Go to the *Administration* -> *Custom Resource Definitions* page, and click *LokiStack*.
```

. On the *Custom Resource Definition Details* page, click *Instances*.

> . Click the options menu {kebab} next to the instance, then click *Delete LokiStack*.

Reviewer: ```suggestion
. Click the options menu {kebab} next to the instance, and then click *Delete LokiStack*.
```

> . Navigate to the *Administration* -> *Custom Resource Definitions* page.

Reviewer: ```suggestion
. Go to the *Administration* -> *Custom Resource Definitions* page.
```

> . Click the options menu {kebab} next to *LokiStack* and select *Delete Custom Resource Definition*.

Reviewer: ```suggestion
. Click the options menu {kebab} next to *LokiStack*, and select *Delete Custom Resource Definition*.
```

. Delete the object storage secret.

> . Navigate to the *Operators* -> *Installed Operators* page.

Reviewer: ```suggestion
. Go to the *Operators* -> *Installed Operators* page.
```

> . Click the options menu {kebab} next to the {loki-op}, then click *Uninstall Operator*.

Reviewer: ```suggestion
. Click the options menu {kebab} next to the {loki-op}, and then click *Uninstall Operator*.
```

. Optional: Delete the `openshift-operators-redhat` project.
+
[IMPORTANT]
====
Do not delete the `openshift-operators-redhat` project if other global Operators are installed in this namespace.
====

.. Switch to the *Home* -> *Projects* page.

> .. Click the options menu {kebab} next to the *openshift-operators-redhat* project, then click *Delete Project*.

Reviewer: ```suggestion
.. Click the options menu {kebab} next to the *openshift-operators-redhat* project, and then click *Delete Project*.
```

> .. Confirm the deletion by typing `openshift-operators-redhat` in the dialog box. Click *Delete*.

Reviewer: ```suggestion
.. Confirm the deletion by typing `openshift-operators-redhat` in the dialog box, and then click *Delete*.
```
